text_dedup/utils/hashfunc.py

```diff
@@ -4,11 +4,6 @@
 from hashlib import sha256
 
 import xxhash
-from blake3 import blake3
-from xxhash import xxh3_64
-from xxhash import xxh3_64_digest
-from xxhash import xxh3_128
-from xxhash import xxh3_128_digest
```

Comment:
This makes some imports invalid in a couple of files:
`bloom_filter.py`
`ccnet.py`
`exact_hash.py`
`simhash.py`

Reply:
i forgot that I actually used those in some of the code. will fix right away